aerepl

```diff
@@ -1,8 +1,52 @@
 #!/usr/bin/env escript
 %% -*- erlang -*-
 %%! -smp enable -setcookie aerepl_cookie -mnesia debug verbose
+
+
+-export([main/1, cli/0, aerepl/1]).
```

Comment:
Why do you export this functions?

Reply:
The arg parser needs them

Reply:
I have commented this line and got the following error:

```
escript: exception error: undefined function aerepl__escript__1710__832252__385779__2:aerepl/1
  in function  escript:run/2 (escript.erl, line 750)
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1 
  in call from init:do_boot/3 
```

I have then exported `aerepl/1` and it worked fine. Is there a case where `main/1` and `cli/0` must be exported in order for this to work?